### PR TITLE
feat(metabar): Add avatarGroup on learn section

### PR DESCRIPTION
**Pull request by AugustinMauroy**
## Description

Adding the `Avatargroup` component to the `MetaBar` in the learn section is a way of thanking contributors. The implementation work was not very complicated because the learn section had already received the authors with their GitHub pseudonyms.

The change only applies to the Learn section and not to the other sections of the site.

## Validation

* Learn section should have same design but with avatar. 
* All metabar should have correct rendering 

## Related Issues

No related Issues.

### Check List

- [X] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [X] I have run `npx turbo format` to ensure the code follows the style guide.
- [X] I have run `npx turbo test` to check if all tests are passing.
- [X] I have run `npx turbo build` to check if the website builds without errors.
- **NA** I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

**Review by mikeesto — COMMENTED** [review]
I think it looks a little awkward atm as overflow is set to scroll

Screen Shot 2024-03-09 at 9 56 39 am

**Comment by ovflowd** [comment]
Im not sure Im in favour of this change.

**Comment by AugustinMauroy** [comment]
> Im not sure Im in favour of this change.

I'm sorry, Claudio, but could you give us some arguments? Because as a person I find it hard to see what's wrong with this addition.

File: components/withMetaBar.tsx [hunk 1]
```diff
@@ -1,26 +1,48 @@
...
+    }));
+  }
 
   return (
     <MetaBar
       items={{
         'components.metabar.lastUpdated': lastUpdated,
         'components.metabar.readingTime': readingTime.text,
+        ...(enableGithubAvatar && {
+          [`components.metabar.${avatars.length > 1 ? 'authors' : 'author'}`]: (
```
Comment (ovflowd) [inline components/withMetaBar.tsx:40]: I feel that the logic of these Avatars (the mapping, the react component, etc) should be inside a hook or something.

**Comment by ovflowd** [comment]
> > Im not sure Im in favour of this change.
> 
> I'm sorry, Claudio, but could you give us some arguments? Because as a person I find it hard to see what's wrong with this addition.

I'm kinda fine eitherways, just ensure to fix the CSS issues from this PR, there is some odd scrollbar on the avatars stuff.

**Comment by ovflowd** [comment]
cc @AugustinMauroy a gentle poke on you to see if you still want to get this merged? There are some visual issues as pointed by @mikeesto

**Comment by AugustinMauroy** [comment]
@ovflowd I need to rebase my branch. but style issue as been fix you can take a look in preview.

File: components/withMetaBar.tsx (hunk 1 above)
Comment (AugustinMauroy) [reply components/withMetaBar.tsx:40]: I'll take a look to that

**Comment by ovflowd** [comment]
@AugustinMauroy still seems broken to me 

image

**Comment by AugustinMauroy** [comment]
simple question which browser did you use ?

**Comment by ovflowd** [comment]
Chrome / Windows 11

File: components/withMetaBar.tsx (hunk 1 above)
Comment (AugustinMauroy) [reply components/withMetaBar.tsx:40]: I had done something cleaner tell me what do you thinks ?

**Comment by ovflowd** [comment]
image

Still having the issue, I'm not sure if you figured out, but the issue is the style on `AvatarGroup` wrapper, it has `overflow-x: scroll` it should be `overflow-x: auto`

**Comment by ovflowd** [comment]
Also, IMO we should change the amount of avatars we render based on the viewport (small, medium, large) you can use the MediaQuery Hook we have to change that!

**Comment by AugustinMauroy** [comment]
> Also, IMO we should change the amount of avatars we render based on the viewport (small, medium, large) you can use the MediaQuery Hook we have to change that!

I love this idea

**Comment by AugustinMauroy** [comment]
yup I had break point for small screen but not mobile because on mobile avatar group is on bottom of page.
Also I had tested scroll on Safari and Chrome I din't see any strange things I hope it's will be same on our 😂

**Comment by mikeesto** [comment]
FWIW the scroll bars are resolved for me now 👍

**Comment by AugustinMauroy** [comment]
> FWIW the scroll bars are resolved for me now 👍

yessss 👏🏻

**Review by ovflowd — APPROVED** [review]
LGTM! Awesome stuff! Thank you for iterating on the requested changes, Augustin!

**Pull request merged**